benchexec/tools/spf.py

```diff
@@ -15,11 +15,7 @@
 """
 import logging
```

Comment:
Imports need to be fixed:
```suggestion
import benchexec.util as util
import benchexec.tools.jpf
```